.github/workflows/code_review_infra.yaml

```diff
@@ -0,0 +1,47 @@
+name: Code Review Infra
+
+on:
+  pull_request:
+    branches: '**'
+    # Run only if there are at least one change matching the following paths
+    paths:
+      - '.infrastructure/**'
+      - '.github/workflows/code_review_infra.yaml'
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: read    # This is required for actions/checkout
+
+jobs:
+  infrastructure_checks:
+    strategy:
+      matrix:
+        environment: [prod] # add dev enviroment if necessary
+    name: Infra checks
+    runs-on: ubuntu-latest
+    environment: ${{ matrix.environment }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Setup Node.JS
+        uses: ./.github/actions/setup-node
+
+      - name: Install dependencies
+        run: npm ci --audit=false --fund=false
+
```

Comment:
```suggestion
```
I don't think these steps are necessary, I think they are related to the cloudfront function steps you've already removed

Reply:
Ok